arkit_utils/undistort_images/undistort_image.py

```diff
@@ -4,6 +4,17 @@
 import os
 import argparse
 from concurrent.futures import ProcessPoolExecutor
+import math
+
+global undistort_data
+undistort_data = None
+
+class UndistortData:
+    def __init__(self, crop_left, crop_top, crop_width, crop_height):
```

Comment:
i think it'd be better to add type annotation

Reply:
added in [this commit](https://github.com/homee-ai/nerfstudio/pull/9/commits/6bdaac87ae1e101e844e3775af68d8f986dee624)